hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleDiskFs.java

```diff
@@ -129,7 +129,12 @@ public void checkHashesAgainstDiskContents() {
 		try {
 			return bytesHelper.allBytesFrom(pathToContentsOf(fid));
 		} catch (IOException e) {
-			log.error("Error reading '{}' @ {}!", asLiteralString(fid), pathToContentsOf(fid), e);
```

Comment:
@JeffreyDallas Can we think of a case where this `IOException` _would_ mean a serious error?

I wonder if we could just add a node-local property, say `files.skipDiskFsHashCheck`, and override it to `true` when running a migration test?

Reply:
Opened #1005 to track a long term solution.